docker/Dockerfile.ci_hexagon

```diff
@@ -62,8 +62,7 @@ ENV HEXAGON_SDK_ROOT "/opt/qualcomm/hexagon_sdk"
 ENV CLANG_LLVM_HOME /opt/clang-llvm
 ENV LD_LIBRARY_PATH $LD_LIBRARY_PATH:/opt/clang-llvm/lib
 ENV PATH /opt/clang-llvm/bin:$PATH
-ENV HEXAGON_TOOLCHAIN "${HEXAGON_SDK_ROOT}/tools/HEXAGON_Tools/8.5.08/Tools"
-ENV HEXAGON_GTEST "${HEXAGON_SDK_ROOT}/utils/googletest/gtest"
```

Comment:
We are not using SDK_PATH anymore. also why do you remove HEXAGON_GTEST here?

Reply:
Decided to remove all reference to `HEXAGON_GTEST` environment variable.  Now using just the `USE_HEXAGON_GTEST` cmake flag. 

You said "not using SDK_PATH" anymore, did you mean "not using SDK_ROOT"?  I am not sure how the `HEXAGON_TOOLCHAIN` env variable ended up using "ROOT" instead of "PATH".  Might have been a copy/paste error on my part?

Reply:
we have removed HEXAGON_SDK_PATH and refactored with HEXAGON_SDK_ROOT. So please use HEXAGON_SDK_PATH from now on. If you still see some variable with name PATH, because https://github.com/apache/tvm/pull/11282/files is waiting for image update to merge.

Regarding HEXAGON_GTEST, now HEXAGON_GTEST is part of the new image that will be merged soon. If you want to change its name we need another image update to be able to access it in the docker. I suggest we keep using the same name until next refactor. Alternative is to manually set it like before

Reply:
1) I will use "ROOT"
2) I don't want to change the name of `HEXAGON_GTEST` environment variable, I am removing it completely.